ttnn/cpp/ttnn/operations/experimental/transformer/nlp_create_qkv_heads/nlp_create_qkv_heads.hpp

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "device/nlp_create_qkv_heads_device_operation.hpp"
+#include "ttnn/run_operation.hpp"
+#include "ttnn/operations/core/core.hpp"
+
+namespace ttnn {
+namespace operations::experimental::transformer {
+
+struct NlpCreateHeadsOperation {
+    static std::tuple<ttnn::Tensor, ttnn::Tensor, ttnn::Tensor> operator() (
+        uint8_t queue_id,
+        const Tensor& input_tensor_q,
+        const std::optional<Tensor>& input_tensor_kv,
+        const uint32_t num_q_heads,
+        const std::optional<uint32_t> num_kv_heads,
+        const bool transpose_k_heads,
+        const std::optional<MemoryConfig>& memory_config,
+        std::optional<std::vector<std::optional<Tensor>>> optional_output_tensors = std::nullopt);
+
+    static std::tuple<ttnn::Tensor, ttnn::Tensor, ttnn::Tensor> operator() (
+        const Tensor& input_tensor_q,
+        const std::optional<Tensor>& input_tensor_kv,
+        const uint32_t num_q_heads,
+        const std::optional<uint32_t> num_kv_heads,
+        const bool transpose_k_heads,
+        const std::optional<MemoryConfig>& memory_config,
+        std::optional<std::vector<std::optional<ttnn::Tensor>>> optional_output_tensors = std::nullopt);
+};
+} // namespace operations::experimental::transformer
+
+namespace experimental::transformer {
+
+constexpr auto nlp_create_qkv_heads = ttnn::register_operation_with_auto_launch_op<
+    "ttnn::experimental::transformer::nlp_create_qkv_heads",
```

Comment:
```suggestion
    "ttnn::experimental::nlp_create_qkv_heads",
```

Reply:
when we register, we register in global namespace like ttnn or ttnn::experimental, unless absolutely necessary to isolate

Reply:
Done